tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,65 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def _add_host_label(instance):
+    """
+    Method add multiple labels to cephadmin node
+    Args:
+        instance: cluster instance
+    Returns:
+        1 -> Fail
+        0 -> Pass
+    """
+
+    node = instance.cluster.get_nodes()[0].hostname
+    cmd = (
+        "cephadm shell ceph orch host add {} node-exporter ".format(node)
+        + "crash osd installer grafana prometheus alertmanager"
+    )
+    try:
+        out, _ = instance.installer.exec_command(cmd, sudo=True)
+        return 0
+    except Exception as e:
+        log.error(e)
+
+
+def run(ceph_cluster, **kw):
```

Comment:
Add description to function explaining what exactly expected here

Reply:
Updated.